### PR TITLE
TypeScript version of Contoso Product

### PR DESCRIPTION
**Pull request by rabwill**

### CODE REVIEW

**Comment by rabwill** [comment]
hey @garrytrinder this is ready to be reviewed and merged. CC @BobGerman

**Review by garrytrinder — CHANGES_REQUESTED** [review]
Apologies for the delay on this @rabwill, 

Thanks for making the changes in from the previous comments, there are a few things that I think we need to address before we merge.

I was able to get this working fine in Teams, but when using it in Copilot, I was unable to go through the authentication flow (when I click on the `Sign in` button) it does nothing.

File: samples/msgext-product-support-sso-ts/infra/botRegistration/azurebot.bicep
[the remark this answers is not in the document]
Comment (rabwill) [reply samples/msgext-product-support-sso-ts/infra/botRegistration/azurebot.bicep:1]: @garrytrinder can you check if this updated version is what you meant by whitespace consistency? Thanks

File: samples/msgext-product-support-sso-ts/infra/azure.parameters.local.json
[the remark this answers is not in the document]
Comment (rabwill) [reply samples/msgext-product-support-sso-ts/infra/azure.parameters.local.json:1]: @garrytrinder can you check if this updated version is what you meant by whitespace consistency? Thanks

File: samples/.DS_Store
[the remark this answers is not in the document]
Comment (rabwill) [reply samples/.DS_Store:1]: I will need some help I cannot find this in my local :( @garrytrinder is it possible to remove this while merging?

**Comment by rabwill** [comment]
> Apologies for the delay on this @rabwill,
> 
> Thanks for making the changes in from the previous comments, there are a few things that I think we need to address before we merge.
> 
> I was able to get this working fine in Teams, but when using it in Copilot, I was unable to go through the authentication flow (when I click on the `Sign in` button) it does nothing.

Thanks @garrytrinder I have addressed most of the issue you have raised except testing in copilot. I need to repro this before I can fix the issue. Thanks for your patience!

**Review by garrytrinder — APPROVED** [review]

**Pull request merged**